xml/System.Reflection.Emit/DynamicMethod.xml

```diff
@@ -1842,7 +1842,7 @@
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- In addition to the listed exceptions, the calling code should be prepared to catch any exceptions thrown by the dynamic method.  
+ In addition to the listed exceptions, the calling code should be prepared to catch any exceptions thrown by the dynamic method. Starting in .NET 7, the originating exception is always wrapped in a <xref:System.Reflection.TargetInvocationException>.
```

Comment:
7.0 didn't change whether TIE is thrown or not, just some edge cases around constructors and byref-like types.